way_to_home/utils/easy_way.py

```diff
@@ -0,0 +1,58 @@
+"""
+    This module use for parsing data got in real time from 'Easy Way' source
+    in GTFS format.
+"""
+import json
+from google import protobuf  # pylint: disable=no-name-in-module, unused-import
+from google.transit import gtfs_realtime_pb2
+
+
+def compile_file(file_gtfs):
+    """ This function compile gtfs file to json file """
+    feed = gtfs_realtime_pb2.FeedMessage()
+
+    try:
+        with open(file_gtfs, 'rb') as file:
+            content = file.read()
+    except FileNotFoundError:
+        return False
```

Comment:
I think if file not found you should call the function that downloads a file from URL

Reply:
I think it is a bad idea. For example, if file not found u will download a file and then u need to again call this function recursive for compile or put it `while True` loop and break if a file was found. The second problem can be that file can be not only downloaded.
So I guess this function just need to compile file if the file was found. In other cases u got False, then u need to create/download/moved/etc file and call the function again.
But it is only my opinion.

Reply:
@Kardan1123 yes, u are right, the same situation with the recursive approach.